gst/gstvalue.h

```diff
@@ -529,17 +541,17 @@ gboolean	gst_value_fraction_multiply	(GValue		*product,
 						 const GValue	*factor1,
 						 const GValue	*factor2);
 gboolean 	gst_value_fraction_subtract (GValue * dest,
-					     const GValue * minuend, 
```

Comment:
this lines seems to be a gst-indent violation

Reply:
I did not change it, stupid gst-indent did.

Reply:
I thought gst-indent doesn't check headers, or it does?